kse/src/main/resources/org/kse/gui/dialogs/resources_de.properties

```diff
@@ -408,7 +408,7 @@ DViewCrl.jdnIssuer.tooltip                  = Eindeutiger Name des CRL Ausstelle
 DViewCrl.jlEffectiveDate.text               = G\u00FCltigkeitsdatum:
 DViewCrl.jlIssuer.text                      = Aussteller:
 DViewCrl.jlNextUpdate.text                  = N\u00E4chste Aktualisierung:
-DViewCrl.jlRevokedCerts.text                = Gesperrte Zertifikate ({0,choice,0#0 Eintr\u00E4ge|1#1 Eintrag|1<{0,number,integer} Eintr\u00E4ge}):
+DViewCrl.jlRevokedCerts.text                = {0,choice,0#Leere CRL (keine Eintr\u00E4ge).|1#Ein Gesperrtes Zertifikat:|1<Gesperrte Zertifikate ({0,number,integer} Eintr\u00E4ge):}
```

Comment:
The "g" for "Ein gesperrtes ..." has to be lower case:
```suggestion
DViewCrl.jlRevokedCerts.text                = {0,choice,0#Leere CRL (keine Eintr\u00E4ge).|1#Ein gesperrtes Zertifikat:|1<Gesperrte Zertifikate ({0,number,integer} Eintr\u00E4ge):}
```

Reply:
I will fix that. :-)